src/Sulu/Bundle/MediaBundle/Controller/MediaController.php

```diff
@@ -287,23 +288,25 @@ private function getListBuilder(Request $request, array $fieldDescriptors, array
             $systemCollection = $this->collectionRepository
                 ->findCollectionByKey(SystemCollectionManagerInterface::COLLECTION_KEY);
 
-            $lftExpression = $listBuilder->createWhereExpression(
-                $fieldDescriptors['lft'],
-                $systemCollection->getLft(),
-                ListBuilderInterface::WHERE_COMPARATOR_LESS
-            );
-            $rgtExpression = $listBuilder->createWhereExpression(
-                $fieldDescriptors['rgt'],
-                $systemCollection->getRgt(),
-                ListBuilderInterface::WHERE_COMPARATOR_GREATER
-            );
+            if ($systemCollection instanceof CollectionInterface) {
```

Comment:
I think this is unexpected to be null and more indicate that Sulu or your Setup was false.

`Assert::notNull($systemCollection, 'The system collection fixtures should have been loaded');`

would be the better fit.